app/ui/src/app/store/integration-support.service.ts

```diff
@@ -73,13 +73,17 @@ export class IntegrationSupportService {
     return this.http.get(url);
   }
 
-  exportIntegration(id: string): Observable<Response> {
-    const url =
-      this.config.getSettings().apiEndpoint +
-      '/integrations/' +
-      id +
-      '/export.zip';
-    return this.http.get(url, { responseType: ResponseContentType.Blob });
+  exportIntegration(...ids: string[]): Observable<Response> {
+    const params = new URLSearchParams();
```

Comment:
Same as the `Http` class, the `URLSearchParams` is deprecated. You should use the `HttpParamsOptions` class instead and inject it from the constructor rather than instantiating it directly.

Reply:
Please submit a PR on to of this one to Convert to newer API.  This PR was not about upgrading usage of older APIs after all.